### PR TITLE
update correct answer: A security engineer is working with a company …

### PR DESCRIPTION
**Pull request by rurychen**
…to design an ecommerce application.

should be ACE

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -871,12 +871,12 @@ Manager Agent on the ECS container instances. Run an inventory report.
 
 ### A security engineer is working with a company to design an ecommerce application. The application will run on Amazon EC2 instances that run in an Auto Scaling group behind an Application Load Balancer (ALB). The application will use an Amazon RDS DB instance for its database. The only required connectivity from the internet is for HTTP and HTTPS traffic to the application. The application must communicate with an external payment provider that allows traffic only from a preconfigured allow list of IP addresses. The company must ensure that communications with the external payment provider are not interrupted as the environment scales. Which combination of actions should the security engineer recommend to meet these requirements? (Choose THREE)
 
-- [ ] Deploy a NAT gateway in each private subnet for every Availability Zone that is in use.
+- [x] Deploy a NAT gateway in each private subnet for every Availability Zone that is in use.
 - [ ] Place the DB instance in a public subnet.
-- [ ] Place the DB instance in a private subnet.
+- [x] Place the DB instance in a private subnet.
 - [x] Configure the Auto Scaling group to place the EC2 instances in a public subnet.
```
Comment (danieldanielecki) [inline README.md:877]: can you please exchange D for E?

**Comment by danieldanielecki** [comment]
@rurychen just one finding, might you please mark E instead of D?

Also, the test is still under a review :) would you like to review other questions?
The test is checked until and including question `You work at a company that makes use of IAM resource...`. That's a question number 60 on line 465.

**Comment by rurychen** [comment]
> @rurychen just one finding, might you please mark E instead of D?
> 
> Also, the test is still under a review :) would you like to review other questions? The test is checked until and including question `You work at a company that makes use of IAM resource...`. That's a question number 60 on line 465.


### You work at a company that makes use of IAM resources. One of the key security policies is to ensure that all data i encrypted both at rest and in transit. Which of the following is one of the right ways to implement this.

- [x] Use S3 SSE and use SSL for data in transit
- [ ] SSL termination on the ELB
- [ ] Enabling Proxy Protocol
- [ ] Enabling sticky sessions on your load balancer

A seems good for me . 
https://docs.aws.amazon.com/AmazonS3/latest/userguide/UsingEncryption.html
and
https://docs.aws.amazon.com/AmazonS3/latest/userguide/security-best-practices.html

**Comment by danieldanielecki** [comment]
merged & yes, because this question was checked by me before a release. Please ping me if you'd like to contribute more!

**Comment by danieldanielecki** [comment]
@rurychen you can connect with me on our Discord: https://discord.gg/RFjtXKfJy3 or my LinkedIn: https://www.linkedin.com/in/danieldanielecki

**Comment by danieldanielecki** [comment]
PS. I also use WeChat ;-) https://danieldanielecki.com you can find my WeChat ID

**Pull request merged**